app/src/examples/ReducedMotionExample.tsx

```diff
@@ -1,61 +1,236 @@
+  },
+  {
+    animation: withRepeat(withTiming(toValue, { duration }), 4, true),
+    text: 'withRepeat (even)',
+  },
+  {
+    animation: withRepeat(withTiming(toValue, { duration }), 3, true),
+    text: 'withRepeat (odd)',
+  },
+  {
+    animation: withRepeat(withTiming(toValue, { duration }), 4, false),
+    text: 'withRepeat\n(no reverse)',
+  },
+];
+
+const CONFIG_EXAMPLES = [
+  {
+    animation: withTiming(toValue, { reduceMotion: 'always', duration }),
+    text: 'always\nreduce',
+  },
+  {
+    animation: withTiming(toValue, { reduceMotion: 'never', duration }),
+    text: 'never\nreduce',
+  },
+  {
+    animation: withTiming(toValue, { reduceMotion: 'system', duration }),
+    text: 'system\nreduce',
+  },
+  {
+    animation: withSequence(
+      'always',
+      withTiming(initialValue + (toValue - initialValue) / 3, { duration }),
+      withTiming(initialValue + (2 * (toValue - initialValue)) / 3, {
+        reduceMotion: 'system',
+        duration,
+      }),
+      withTiming(toValue, { reduceMotion: 'never', duration })
+    ),
+    text: 'nested',
+  },
+];
+
+const EXAMPLES = [
+  {
+    title: 'Simple',
+    component: (
+      <>
+        <HookExample />
+        {mapExamples(SIMPLE_EXAMPLES)}
+      </>
+    ),
+  },
+  {
+    title: 'Repeat',
+    component: mapExamples(REPEAT_EXAMPLES),
+  },
+  {
+    title: 'Decay',
+    component: <DecayExample />,
+  },
+  {
+    title: 'Config',
+    component: mapExamples(CONFIG_EXAMPLES),
+  },
+];
+
+export default function App() {
+  const [currentExample, setCurrentExample] = useState(0);
+
+  return (
+    <View style={styles.container}>
+      {EXAMPLES.map((x, i) => (
+        <Button key={i} onPress={() => setCurrentExample(i)} title={x.title} />
+      ))}
+      {EXAMPLES[currentExample].component}
+    </View>
+  );
+}
+
+function HookExample() {
+  const sv = useSharedValue(initialValue);
   const shouldReduceMotion = useReducedMotion();
 
-  const hookExampleStyle = useAnimatedStyle(() => ({
-    transform: [{ translateX: shouldReduceMotion ? 0 : hookExample.value }],
+  const animatedStyle = useAnimatedStyle(() => ({
+    transform: [{ translateX: shouldReduceMotion ? 0 : sv.value }],
   }));
 
   React.useEffect(() => {
-    hookExample.value = withRepeat(
-      withTiming(-hookExample.value, {
-        duration,
-      }),
-      -1,
-      true
+    sv.value = withRepeat(withTiming(toValue, { duration }), -1, true);
+  });
+
+  return (
+    <Animated.View style={[styles.box, animatedStyle]}>
+      <Text style={styles.text}>useReducedMotion</Text>
+    </Animated.View>
+  );
+}
+
+function mapExamples(
+  examples: {
+    animation: number;
+    text: string;
+  }[]
+) {
+  return examples.map((example, i) => {
+    return (
+      <Example key={i} animation={example.animation} text={example.text} />
     );
   });
+}
+
+function Example(props: { animation: number; text: string }) {
+  const sv = useSharedValue(initialValue);
```

Comment:
Let's add a newline here:
```suggestion
  const sv = useSharedValue(initialValue);

```